### ISSUE TITLE
Swapping between two images.

### ISSUE THREAD

**Issue by AIdeveloper-oz** [AliaksandrSiarohin/motion-cosegmentation#17]
Hi, I am a great fan for this kinda works and my appreciations to you. On analyzing the code, I get to know that, you swap face from an image to the face in a video feed. I have seen image to image face swapping using 68/81 facial keypoints. The results of your work seem to be better than keypoint based method. Is there any implementation of segmentation based swapping between images? Help, if you could. 
Thanks.

**Comment by AliaksandrSiarohin** [comment]
What do you mean a segmentation based swapping? This work uses segmentation.

**Comment by AIdeveloper-oz** [comment]
Yeah, I am aware of that. I referred **Segmentation based swapping ** to your work only. I found you are doing it between videos, where frames are played with. I simply wanted to try this between two images. (src image, target image -> segmentation->copy the src mask on target image. Maybe this is simple and I am a new bee. Help, if you could.
 Thanks.

**Comment by AliaksandrSiarohin** [comment]
You can just use a single frame video, this will be effectively an image base swap.

**Comment by instant-high** [comment]
You can also use a *.gif as target_video (instead of *.mp4) and as result_video use also *.gif

Then you have to change the following lines in part_swap.py

(241) fps = reader.get_meta_data()['fps']
 --> #fps = reader.get_meta_data()['fps']

(244) imageio.mimsave(opt.result_video, [img_as_ubyte(frame) for frame in predictions], fps=fps, bitrate=bitrate)
    -->imageio.mimsave(opt.result_video, [img_as_ubyte(frame) for frame in predictions])

**Comment by cbarne10** [comment]
Hi - I am attempting this with a single frame video as well. The full facial swap doesn't seem to be consistent or scaled. Sometimes, certain features are not swapped (i.e. a smile). I am performing it on supervised with the vox 15 segments checkpoint. Any suggestions to what may be occurring. Thanks!
